cli/src/test/java/org/commonjava/maven/ext/cli/CliTest.java

```diff
@@ -142,17 +143,18 @@ public void checkLocalRepositoryWithDefaults() throws Exception
     public void checkLocalRepositoryWithDefaultsAndModifiedUserSettings() throws Exception
     {
         boolean restore = false;
-        Path source = Paths.get ( System.getProperty( "user.home" ) + File.separatorChar + ".m2" + File.separatorChar + "settings.xml");
+        Path source = Paths.get( System.getProperty( "user.home" ) + File.separatorChar + ".m2" + File.separatorChar
+                                                 + "settings.xml" );
         Path backup = Paths.get( source.toString() + '.' + UUID.randomUUID().toString() );
-        Path tmpSettings = Paths.get ( getClass().getResource("/settings-test.xml").toURI() );
+        Path tmpSettings = Paths.get( getClass().getResource( "/settings-test.xml" ).toURI() );
 
         try
         {
             if ( source.toFile().exists() )
             {
-                System.out.println ("Backing up settings.xml to " + backup);
+                System.out.println( "Backing up settings.xml to " + backup );
```

Comment:
Logger should be used.

Reply:
Doesn't really matter as its in a test and the output is muted anyway inside the systemoutrule.

Reply:
Muted? So where can you see the message at all, then?

Reply:
If the test fails :-)

Reply:
I could just remove it, but it was preexisting code and I hadn't changed it.